test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -275,6 +275,7 @@ public void TestLoadDatetimeColumnWithNullValue()
         private DatabaseSource GetIrisDatabaseSource(string command, int commandTimeoutInSeconds = 30)
         {
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+#pragma warning disable CS0618 // 'SqlClientFactory' is obsolete: 'Use the Microsoft.Data.SqlClient package instead.'
```

Comment:
I didn't see this build failure in the previous CI run, but I did hit it in my local machine. If you prefer that I remove these pragmas, let me know, @michaelgsharp .